app/experimenter/nimbus-ui/src/components/FormattedJson/index.tsx

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+
+export const FormattedJson = ({ json }: { json: string }) => (
+  <code>
+    <pre className="mb-0">{json}</pre>
```

Comment:
Oh! So this isn't _formatting_ the JSON, it's just a `<pre>` wrapper for pre-formatted JSON strings?

Reply:
`<code>` and `<pre>`, right, though it does transform strings with `\n` into exactly what the output is. Could use a different name if there's a better one 🤔 

We may need another issue if we want long full targeting expressions or feature configs to format like the `recipeJson`. Check out what's returned by the server between `recipeJson` and other values: https://experimenter.services.mozilla.com/nimbus/howdy-doody